docs/source/changes.rst

```diff
@@ -5,6 +5,47 @@
 Changes
 =======
 
+v0.5.0
+======
+
+**Breaking change**
+
+* Remove the ``BoxCox`` transform. ( `#222`_ )
+
+**New features**
+
+* Add new contribution ``iSMD`` by `qi`_. ( `#208`_ )
+* Add classification support for ``Trainer``. ( `#184`_ )
+
```

Comment:
More new features:
* Add ``IQSPR4DF`` to support ``pandas.DataFrame`` input to iQSPR.
* Add ``LayeredFP``, ``PatternFP``, and ``MHFP`` (new rdkit fingerprints).

Reply:
Are these two have the issue numbers? If they have, I'd like to put the issue number and links at the end as others did.

Reply:
I am not sure. The one that I know, I already put... sorry, hard to trace back now.